website_cookie_notice/__openerp__.py

```diff
@@ -21,20 +21,19 @@
 {
     'name': 'Cookie notice',
     'summary': 'Show cookie notice according to cookie law',
-    'version': '8.0.1.0.0',
+    'version': '9.0.2.1.0',
```

Comment:
@elicoidal see https://github.com/OCA/website/blob/8.0/website_cookie_notice/__openerp__.py#L24
I just increased the odoo version number